src/api/commonFetch.ts

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
```

Comment:
This function looks almost exactly as `fetchNews`. Please update `fetchNews` and just add the `category` parameter there and remove this file